diff --git a/source/op/CMakeLists.txt b/source/op/CMakeLists.txt
index ddcd833..da57dae 100644
--- a/source/op/CMakeLists.txt
+++ b/source/op/CMakeLists.txt
@@ -30,6 +30,10 @@ if (BUILD_PY_IF)
   target_link_libraries(
     op_grads PRIVATE ${TensorFlowFramework_LIBRARY}
     )
+  find_package(Protobuf)
+  if(Protobuf_LIBRARIES)
+    target_link_libraries(op_abi PRIVATE ${Protobuf_LIBRARIES})
+  endif()
   target_include_directories(op_abi PUBLIC ${TensorFlow_INCLUDE_DIRS})
   target_include_directories(op_grads PUBLIC ${TensorFlow_INCLUDE_DIRS})
   if (APPLE)
