From 5d6b33423f41e62b2355adcf2d49fb0c956b6094 Mon Sep 17 00:00:00 2001                                             
From: Jinzhe Zeng <jinzhe.zeng@rutgers.edu>
Date: Sun, 3 Mar 2024 06:30:15 -0500
Subject: [PATCH] fix libtorch not found in $RPATH

Signed-off-by: Jinzhe Zeng <jinzhe.zeng@rutgers.edu>
---
 source/api_cc/CMakeLists.txt | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/source/api_cc/CMakeLists.txt b/source/api_cc/CMakeLists.txt
index cd42594f..2802498b 100644
--- a/source/api_cc/CMakeLists.txt
+++ b/source/api_cc/CMakeLists.txt
@@ -33,8 +33,10 @@ if(Protobuf_LIBRARY)
 endif()
 
 set_target_properties(
-  ${libname} PROPERTIES INSTALL_RPATH "$ORIGIN;${TensorFlow_LIBRARY_PATH}"
-                        BUILD_RPATH "$ORIGIN/../op")
+  ${libname}
+  PROPERTIES INSTALL_RPATH "$ORIGIN;${TensorFlow_LIBRARY_PATH}"
+             INSTALL_RPATH_USE_LINK_PATH TRUE
+             BUILD_RPATH "$ORIGIN/../op")
 target_compile_definitions(${libname} PRIVATE TF_PRIVATE)
 if(CMAKE_TESTING_ENABLED)
   target_link_libraries(${libname} PRIVATE coverage_config)
-- 
2.43.0
