diff --git a/setup.py b/setup.py
index 4803b043..34ebc91d 100644
--- a/setup.py
+++ b/setup.py
@@ -135,6 +135,7 @@ setup(
         "-DBUILD_PY_IF:BOOL=TRUE",
         "-DBUILD_CPP_IF:BOOL=FALSE",
         *cmake_args,
+        *filter(lambda x: not x.startswith("-DCMAKE_INSTALL_PREFIX="), os.environ.get("CMAKE_ARGS", "").split()),
     ],
     cmake_source_dir="source",
     cmake_minimum_required_version="3.0",
