From d7b7d52eadc02c1a8c41edeb2820e48968213859 Mon Sep 17 00:00:00 2001
From: Jinzhe Zeng <jinzhe.zeng@rutgers.edu>
Date: Tue, 31 Aug 2021 03:51:19 -0400
Subject: [PATCH 2/2] use @loader_path on macos instead of $ORIGIN

---
 source/op/CMakeLists.txt | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/source/op/CMakeLists.txt b/source/op/CMakeLists.txt
index 07d030ef..96c02ec7 100644
--- a/source/op/CMakeLists.txt
+++ b/source/op/CMakeLists.txt
@@ -32,6 +32,20 @@ if (BUILD_PY_IF)
     )
   target_include_directories(op_abi PUBLIC ${TensorFlow_INCLUDE_DIRS})
   target_include_directories(op_grads PUBLIC ${TensorFlow_INCLUDE_DIRS})
+  if (APPLE)
+    set_target_properties(
+      op_abi 
+      PROPERTIES 
+      COMPILE_FLAGS ${OP_CXX_FLAG}
+      INSTALL_RPATH @loader_path
+      )
+    set_target_properties(
+      op_grads
+      PROPERTIES
+      COMPILE_FLAGS ${OP_CXX_FLAG}
+      INSTALL_RPATH @loader_path
+      )
+  else()
   set_target_properties(
     op_abi 
     PROPERTIES 
@@ -44,6 +58,7 @@ if (BUILD_PY_IF)
     COMPILE_FLAGS ${OP_CXX_FLAG}
     INSTALL_RPATH $ORIGIN
     )
+  endif ()
 endif (BUILD_PY_IF)
 
 if (BUILD_CPP_IF)
-- 
2.31.1

