diff --git a/source/lmp/plugin/CMakeLists.txt b/source/lmp/plugin/CMakeLists.txt
index f6ce774..5762d8a 100644
--- a/source/lmp/plugin/CMakeLists.txt
+++ b/source/lmp/plugin/CMakeLists.txt
@@ -75,6 +75,9 @@ if (DEFINED LAMMPS_SOURCE_ROOT OR DEFINED LAMMPS_VERSION)
     PRIVATE "LMPPLUGIN" # fix header path
     PRIVATE "LAMMPS_VERSION_NUMBER=${LAMMPS_VERSION_NUMBER}"
   )
+  if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
+    set_target_properties(${libname} PROPERTIES LINK_FLAGS "-Wl,-undefined,dynamic_lookup")
+  endif()
 
   install(TARGETS ${libname} DESTINATION lib/)
 
