diff --git a/source/ipi/CMakeLists.txt b/source/ipi/CMakeLists.txt
index d71a904..b23b65e 100644
--- a/source/ipi/CMakeLists.txt
+++ b/source/ipi/CMakeLists.txt
@@ -16,6 +16,14 @@ add_executable(${ipiname} ${DRIVER_SOURCE_FILES})
 target_link_libraries(${ipiname} PRIVATE ${libipiname} ${LIB_DEEPMD_CC}${variant_name})
 target_include_directories(${ipiname} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../3rdparty/)
 
+if (APPLE)
+  set_target_properties(
+    ${ipiname}
+    PROPERTIES
+    INSTALL_RPATH "@loader_path/../lib:${TensorFlow_LIBRARY_PATH}"
+    COMPILE_DEFINITIONS ${prec_def}
+  )
+else()
 set_target_properties(
   ${ipiname}
   PROPERTIES
@@ -23,6 +31,7 @@ set_target_properties(
   INSTALL_RPATH "$ORIGIN/../lib:${TensorFlow_LIBRARY_PATH}"
   COMPILE_DEFINITIONS ${prec_def}
 )
+endif()
 
 install(
   TARGETS	${libipiname}
