diff --git a/setup.py b/setup.py
index 98bcfde..e499d3a 100644
--- a/setup.py
+++ b/setup.py
@@ -3,7 +3,7 @@ from skbuild.exceptions import SKBuildError
 from skbuild.cmaker import get_cmake_version
 from setuptools_scm import get_version
 from packaging.version import LegacyVersion
-from os import path, makedirs
+from os import path, makedirs, getenv
 import os, imp, sys, platform, sysconfig
 
 def get_dp_install_path() :
@@ -71,6 +71,7 @@ setup(
                 '-DFLOAT_PREC:STRING=high',
                 '-DDP_PIP_INSTALL_PATH=%s' % dp_pip_install_path,
                 '-DDP_SETUP_INSTALL_PATH=%s' % dp_setup_install_path,
+                "-DCMAKE_OSX_DEPLOYMENT_TARGET={}".format(getenv("MACOSX_DEPLOYMENT_TARGET")),
     ],
     cmake_source_dir='source',
     cmake_minimum_required_version='3.0',
